Add new codes

Hoeppner1867 opened the pull request:
Updates & Uploads

Hoeppner1867 committed:
Uploads and updates

SicroAtGit commented:
Your local branch "thorsten" was diverged from the current master branch. I fixed that.

SicroAtGit commented:
Do you really want to put your sample image files under GPL 2.0?

A few helpful links:
- https://stackoverflow.com/questions/5753293/gnu-gpl-license-and-images
- https://www.quora.com/What-are-GPL-images-and-what-are-some-examples
- https://opensource.stackexchange.com/questions/4409/media-what-are-the-arguments-for-using-a-software-license
- https://choosealicense.com/non-software/

Hoeppner1867 commented:
Ich habe diese nur genommen, weil die meisten anderen Bilder diese haben.
Wenn du eine bessere weißt, bin ich für alles offen.


---
Ich habe diese Nachricht mit der GMX Mail App für Windows erstellt.



-----Ursprüngliche Nachricht-----Von: "Alexander Bolli" <notifications@github.com>Gesendet: Donnerstag, 29. August 2019 17:09An: SicroAtGit/PureBasic-CodeArchiv-Rebirth <purebasic-codearchiv-rebirth@noreply.github.com>Cc: "Thorsten Hoeppner" <thorsten.hoeppner@gmx.de>; Author <author@noreply.github.com>Betreff: Re: [SicroAtGit/PureBasic-CodeArchiv-Rebirth] Add new codes (#10)</author@noreply.github.com></thorsten.hoeppner@gmx.de></purebasic-codearchiv-rebirth@noreply.github.com>


Do you really want to put your sample image files under GPL 2.0?
A few helpful links:

https://stackoverflow.com/questions/5753293/gnu-gpl-license-and-images
https://www.quora.com/What-are-GPL-images-and-what-are-some-examples
https://opensource.stackexchange.com/questions/4409/media-what-are-the-arguments-for-using-a-software-license
https://choosealicense.com/non-software/

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.

SicroAtGit commented:
Die GPL, die du gewählt hast, ist hauptsächlich für Quellcodes.

Für Dateien, die keinen Quellcode enthalten, kannst du z. B. die Creative Commons Attribution 4.0 International nehmen: https://choosealicense.com/licenses/cc-by-4.0
Ein Beispiel habe ich dir im deutschen PB-Forum per PN geschickt.

---

**Damit ich dein Branch nicht jedes Mal reparieren muss:**

Bitte lösche deinen lokalen Branch "thorsten" in GitHub Desktop:

- Current Branch "thorsten": Branch => Delete...
(In "Yes, delete this branch on the remote" darf kein Häkchen sein)

Erstelle den lokalen Branch neu:

- Auf "Current branch" klicken und "origin/thorsten" auswählen.

Danach sollte dein lokaler Branch wieder in Ordnung sein.

---

Noch eine Bitte: Schreib hier in GitHub besser alles in Englisch. Es gibt Mitleser, die kein deutsch verstehen und so bleibt die Sprache hier einheitlich.

Hoeppner1867 committed:
Change license text

SicroAtGit committed:
Text files reformatted

- Added a blank line at the end of the file
- Lines wrapped after 80 characters
- Removed trailing spaces

SicroAtGit committed:
Fix missing 'CompilerEndIf'

SicroAtGit commented on Date/Date64Module.pbi:
```diff
@@ -0,0 +1,741 @@
+﻿;   Description: Support for enlarged date range (64 bit unix timestamp)
```
The "official" version of the module already exists in the archive. Your version only uses a different programming style and removes some informative comments. Please accept programming styles from others and don't be angry that I won't include this module.

SicroAtGit committed:
CalendarModule: Change the inclusions to already existing modules

SicroAtGit committed:
Remove duplicate or no longer needed codes

SicroAtGit reviewed (dismissed):
I've adjusted a few things.
Do you agree with the changes?

Hoeppner1867 commented:
CanvasTooltipModule and TooltipExModule are not the same.


---
Ich habe diese Nachricht mit der GMX Mail App für Windows erstellt.



-----Ursprüngliche Nachricht-----Von: "Alexander Bolli" <notifications@github.com>Gesendet: Samstag, 31. August 2019 20:04An: SicroAtGit/PureBasic-CodeArchiv-Rebirth <purebasic-codearchiv-rebirth@noreply.github.com>Cc: "Thorsten Hoeppner" <thorsten.hoeppner@gmx.de>; Author <author@noreply.github.com>Betreff: Re: [SicroAtGit/PureBasic-CodeArchiv-Rebirth] Add new codes (#10)</author@noreply.github.com></thorsten.hoeppner@gmx.de></purebasic-codearchiv-rebirth@noreply.github.com>


@SicroAtGit approved this pull request.
I've adjusted a few things.Do you agree with the changes?
—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.

SicroAtGit committed:
CalendarModule: Fix inclusion of a wrong module

The module "TooltipExModule" was wrongly included instead of the module
"CanvasTooltipModule".

SicroAtGit reviewed (approved):
Thanks, I didn't notice it.
Is now everything ok?

Hoeppner1867 commented:
I think so.


---
Ich habe diese Nachricht mit der GMX Mail App für Windows erstellt.



-----Ursprüngliche Nachricht-----Von: "Alexander Bolli" <notifications@github.com>Gesendet: Samstag, 31. August 2019 22:56An: SicroAtGit/PureBasic-CodeArchiv-Rebirth <purebasic-codearchiv-rebirth@noreply.github.com>Cc: "Thorsten Hoeppner" <thorsten.hoeppner@gmx.de>; Author <author@noreply.github.com>Betreff: Re: [SicroAtGit/PureBasic-CodeArchiv-Rebirth] Add new codes (#10)</author@noreply.github.com></thorsten.hoeppner@gmx.de></purebasic-codearchiv-rebirth@noreply.github.com>


@SicroAtGit approved this pull request.
Thanks, I didn't notice it.Is now everything ok?
—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.
